src/gui_common/DraggableScrollContainer.cs

```diff
@@ -82,7 +82,7 @@ public override void _Ready()
     {
         base._Ready();
 
-        ContentPath ??= GetChild(0).GetPath();
+        ContentPath ??= GetChild(2).GetPath();
```

Comment:
Why this change, is it a bugfix ?

Reply:
This seems very suspicious to me, and unless the documentation etc. is changed elsewhere this is an incorrect change as this makes the comments and code behaviour different. Also it's very unexpected that this would pick the *third* child as the content if the path is not set.

Reply:
The index definitely shouldn't be any other than 0. ScrollContainers the way it's designed should only have one Control child.

Reply:
Child 0 is the HScroll bar; 1 VScroll

Reply:
Ah that makes sense, this should be documented then.